flexget/plugins/sites/lostfilm.py

```diff
@@ -95,67 +95,58 @@ def on_task_input(self, task, config):
                     log.exception('Could not parse %s, probably HTML markup was changed', release_url)
                     continue
 
-                # TODO We cannot determine available qualities right now since they are listed on the download page.
-                # So, assume all possible qualities are available and emit entries for them
-                for quality_code, quality_params in QUALITY_MAP.items():
-                    quality, file_ext = quality_params
+
+                torrents = self.get_torrents(task, lostfilm_num, season_num, episode_num)
+                for quality, torrent_link in torrents.items():
                     new_entry = Entry()
-                    new_entry['url'] = "{}?{}".format(V_SEARCH_URL, urllib.urlencode({'c': lostfilm_num, 's': season_num, 'e': episode_num}))
+                    new_entry['title'] = '{:s} S{:02d}E{:02d} - {:s} [{:s}]'.format(series_name_eng, season_num, episode_num, episode_name_eng, quality)
+                    new_entry['url'] = torrent_link
                     new_entry['name'] = series_name_eng
-                    new_entry['title'] = '{:s} S{:02d}E{:02d} - {:s} [{:s}, {:s}]'.format(series_name_eng, season_num, episode_num, episode_name_eng, quality, file_ext)
                     new_entry['season'] = season_num
                     new_entry['episode'] = episode_num
                     new_entry['quality'] = quality
-                    new_entry['lf_quality_code'] = quality_code
                     new_entry['series_name_rus'] = series_name_rus
                     new_entry['episode_name_rus'] = episode_name_rus
                     new_entry['series_name_eng'] = series_name_eng
                     new_entry['episode_name_eng'] = episode_name_eng
                     entries.append(new_entry)
         return entries
 
-    # urlrewriter API
-    def url_rewritable(self, task, entry):
-        return entry['url'].startswith(V_SEARCH_URL)
-
-    def url_rewrite(self, task, entry):
-        release_url = entry['url']
-
+    def get_torrents(self, task, lostfilm_num, season_num, episode_num):
+        release_url = "{}?{}".format(V_SEARCH_URL, urllib.urlencode({'c': lostfilm_num, 's': season_num, 'e': episode_num}))
```

Comment:
`urllib.urlencode` is not compatible with Python 3. Use this import `from future.moves.urllib.parse import urlencode`

Reply:
Also, you should just give requests the get params like this: http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

Reply:
Done, works

Reply:
`task.requests` call updated